doc/news/changes/minor/20220530KraftKronbichler

```diff
@@ -0,0 +1,3 @@
+Fixed: VectorTools::project can now be instantiated for complex numbers, bypassing undefined operations when compiling the parallel code path with MatrixFree evaluators.
+<br>
+(Pascal Kraft, Martin Kronbichler, 2022/05/30)
```

Comment:
Is Pascal Kraft == SilverLinings89?

Reply:
That's what the github profile suggest, but I'd be happy to be corrected here.

Reply:
I've got no reason to doubt you :-)